tutorials/cloud-controller-manager/index.mdx

```diff
@@ -29,7 +29,7 @@ The Scaleway Cloud Controller Manager is currently under active development and
 
 <Message type="requirement">
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
-  - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)
+  - You have [configured your SSH key](/console/my-project/how-to/create-ssh-key/)
   - You have [created your API Key](/identity-and-access-management/iam/how-to/create-api-keys/)
```

Comment:
Should "key" in API Key be capitalized if SSH key isnt ?

Reply:
Will fix it. Thanks